### PR TITLE
added PDF weighting calling functions

### PR DESCRIPTION
**Pull request by sungbinoh**
Functions to call pdf weights and alpha_s systematics using LHAPDF are added.

### CODE REVIEW

File: Analyzers/src/AnalyzerCore.C [hunk 1]
```diff
@@ -759,6 +759,14 @@ void AnalyzerCore::initializeAnalyzerTools(){
   cfEst.SetDataYear(DataYear);
   cfEst.ReadHistograms();
 
+  TString current_pdf_name = "NNPDF31_nnlo_hessian_pdfas";
```
Comment (sungbinoh) [inline Analyzers/src/AnalyzerCore.C]: To save memory, you need to call PDF maps with strings which are need in the analysis code at initializing step.

Comment (jedori0228) [reply Analyzers/src/AnalyzerCore.C]: It looks like one should modify AnalyzerCore to get different PDF set..? I think we'd better have a separate class for this to make it configurable and orgnaized. Let me try this.

Comment (jedori0228) [reply Analyzers/src/AnalyzerCore.C]: btw, where are the LHAPDF classes?

**Comment by jedori0228** [comment]
Working on #17

**Pull request closed without merging**